README.md

```diff
@@ -11,7 +11,7 @@ This is the EnergyPlus Development Repository.  EnergyPlus™ is a whole buildin
  - Many users of EnergyPlus are active on [Unmet Hours](https://unmethours.com/), so that's a great place to start if you have a question about EnergyPlus or building simulation.
  - For more in-depth, developer-driven support, please utilize the [EnergyPlus Helpdesk](http://energyplus.helpserve.com/).
 
-# Releases [![](https://img.shields.io/badge/latest%20release-8.6.0-orange.svg)](https://github.com/NREL/EnergyPlus/releases/latest)
+# Releases [![](https://img.shields.io/badge/latest%20release-8.8.0-orange.svg)](https://github.com/NREL/EnergyPlus/releases/latest)
```

Comment:
Or, do we want to remove the version number to prevent forgetting this in the future?

Reply:
Yeah, unless we're able to do a dynamic badge there, it's probably better to just take the version out for now.  To keep the version number in there, we could look at getting the latest release number from energyplus.net and using that to build the badge, but that's probably a "later" thing.